brainatlas_api/bg_atlas.py

```diff
@@ -8,6 +8,8 @@
 
 COMPRESSED_FILENAME = "atlas.tar.gz"
 
+__all__ = ["FishAtlas", "RatAtlas", "AllenBrain25Um", "AllenHumanBrain500Um"]
```

Comment:
Probably this could be handled with something like what is decribed here:
https://stackoverflow.com/questions/1796180/how-can-i-get-a-list-of-all-classes-within-current-module-in-python

Putting some conditions on the class name (eg it should not be BGAtlas)

Reply:
Sure we can do that as well, I've done it manually because we don't have that many and they won't be changing too much.